Packages/com.vrchat.ClientSim/Runtime/System/ClientSimPlayerManager.cs

```diff
@@ -675,6 +675,63 @@ public static void SetVoiceGain(VRCPlayerApi player, float value)
         }
 
         #endregion
+        
+        #region Player Scaling
+        
+        private const float AVATAR_MIN_EYE_HEIGHT_DEFAULT = 0.2f;
+        private const float AVATAR_MAX_EYE_HEIGHT_DEFAULT = 5f;
+        private const float AVATAR_EYE_HEIGHT_DEFAULT = 1.6f;
+        
+        public static bool GetManualAvatarScalingAllowed(VRCPlayerApi player)
+        {
+            Debug.Log($"[ClientSim VRCPlayerApi.GetManualAvatarScalingAllowed] Returning default value of true.");
+            return true;
+        }
+
+        public static void SetManualAvatarScalingAllowed(VRCPlayerApi player, bool value)
+        {
+            Debug.Log($"[ClientSim VRCPlayerApi.SetManualAvatarScalingAllowed] called with value {value}, doesn't do anything in ClientSim yet.");
+        }
+        
+        public static float GetAvatarEyeHeightMinimumAsMeters(VRCPlayerApi arg)
+        {
+            Debug.Log($"[ClientSim VRCPlayerApi.GetAvatarEyeHeightMinimumAsMeters] Returning default value of {AVATAR_MIN_EYE_HEIGHT_DEFAULT}.");
+            return AVATAR_MIN_EYE_HEIGHT_DEFAULT;
+        }
+
+        public static float GetAvatarEyeHeightMaximumAsMeters(VRCPlayerApi arg)
+        {
+            Debug.Log($"[ClientSim VRCPlayerApi.GetAvatarEyeHeightMaximumAsMeters] Returning default value of {AVATAR_MAX_EYE_HEIGHT_DEFAULT}.");
+            return AVATAR_MAX_EYE_HEIGHT_DEFAULT;
+        }
+        
+        public static float GetAvatarEyeHeightAsMeters(VRCPlayerApi arg)
+        {
+            Debug.Log($"[ClientSim VRCPlayerApi.GetAvatarEyeHeightAsMeters] Returning default value of {AVATAR_EYE_HEIGHT_DEFAULT}.");
+            return AVATAR_EYE_HEIGHT_DEFAULT;
```

Comment:
I think we can safely return ClientSim's player height for now.

```suggestion
            return ClientSimSettings.Instance.playerHeight;
```